Src/FluentAssertions/Formatting/FormattingOptions.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace FluentAssertions.Formatting
+{
+    public class FormattingOptions
+    {
+        /// <summary>
+        /// Indicates whether the formatter should use line breaks when the <see cref="IValueFormatter"/> supports it.
+        /// </summary>
+        public bool UseLineBreaks { get; set; }
+
+        /// <summary>
+        /// Determines the depth until which the library should try to render an object graph.
+        /// </summary>
+        /// <value>
+        /// A depth of 1 will only the display the members of the root object.
+        /// </value>
+        public int MaxDepth { get; set; } = 5;
+
+        /// <summary>
+        /// Determines the maximum number of lines the library should use to render an object graph.
+        /// </summary>
+        public int MaxLines { get; set; } = 100;
+
+        public FormattingOptions Clone()
+        {
+            return new()
+            {
+                UseLineBreaks = UseLineBreaks,
+                MaxDepth = MaxDepth
```

Comment:
Should an updated `MaxLines` be included in the cloning?

Reply:
Good catch. I'm not done yet though ;-)